### PR TITLE
external API: update docs

### PR DESCRIPTION
**Pull request by holiber**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -202,4 +202,4 @@
     "webpack-manifest-plugin": "2.2.0",
     "webpack-merge": "4.2.1"
   }
-}
\ No newline at end of file
+}
```
Comment (andycreeth) [inline package.json:205]: Why is this change in here?

Comment (holiber) [reply package.json:205]: I had to reinstall `typedock` and then manually removed it from the `package.json`

Comment (holiber) [reply package.json:205]: I bet my IDE applied to autoformatting after that

Comment (andycreeth) [reply package.json:205]: Next time please remove the change before merging.

**Pull request merged**